### ISSUE TITLE
Populate exception message

### ISSUE THREAD

**Issue by hgong-snap** [pytorch/serve#1875]
### 🚀 The feature

Populate exception message

### Motivation, pitch

Currently, if the handler raise some random error, the client will receives only `InternalServerException` without any error/exception details
```
<_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "Prediction failed
org.pytorch.serve.http.InternalServerException"
	debug_error_string = "UNKNOWN:Error received from peer ipv6:%5B::1%5D:17100 {created_time:"2022-09-26T15:40:29.722145-07:00", grpc_status:14, grpc_message:"Prediction failed\norg.pytorch.serve.http.InternalServerException"}"
>
```

what I do now is basically add a wrapper everywhere:
```python
from ts.utils.util import PredictionException

# in handler
try:
  self.do_stuff_and_may_raise_exception()  # this will raise some random Exception
except Exception as e:
  raise PredictionException(str(e), 503)
```
This way exception message from original exception will be correctly populated:
```
<_InactiveRpcError of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "Could not automatically determine credentials. Please set GOOGLE_APPLICATION_CREDENTIALS or explicitly create credentials and re-run the application. For more information, please see https://cloud.google.com/docs/authentication/getting-started
org.pytorch.serve.http.InternalServerException"
```

Is this expected?  is it possible to have this kinda wrapper everywhere?

**Comment by msaroufim** [comment]
Yeah for now the suggestion would be to manually add try blocks around parts of your code that could potentially raise an exception. I think the ask is valid though that whatever exception is thrown by a handler it should get caught with an error code and message by default but just need to think about how to do this.

**Comment by lxning** [comment]
@hgong-snap [Here](https://github.com/pytorch/serve/blob/master/frontend/server/src/main/java/org/pytorch/serve/util/GRPCUtils.java#L46) is the map b/w grpc code and http code defined in Torchserve. I expect the error message from handler is still available for client according to the [code](https://github.com/pytorch/serve/blob/master/frontend/server/src/main/java/org/pytorch/serve/job/GRPCJob.java#L108).

**Comment by msaroufim** [comment]
@hgong-snap did the above answer your question?

**Comment by hgong-snap** [comment]
thanks for the code pointer. but that seems not the case...   seems error detail can be dropped if the exception is not `PredictionException`

**Comment by msaroufim** [comment]
Is the error truncated only on the client end? I'm assuming you see the error correctly populated in logs `logs/model_log.log`?
